cl/citations/tests.py

```diff
@@ -507,23 +507,26 @@ def test_make_html(self):
              'at\n<span class="page">99</span> </span><pre class="inline">'
              '(quoting foo)</pre>'),
 
-            # First kind of id. citation ("Id., at 123")
+            # Id. citation ("Id., at 123")
             ('asdf, id., at 123. Lorem ipsum dolor sit amet',
              '<pre class="inline">asdf, </pre><span class="citation no-link">'
-             'id., at 123. Lorem </span><pre class="inline">ipsum dolor sit '
-             'amet</pre>'),
-
-            # Second kind of id. citation ("Ibid.")
-            ('asdf, Ibid. Lorem ipsum dolor sit amet',
-             '<pre class="inline">asdf, </pre><span class="citation no-link">'
-             'Ibid. Lorem ipsum dolor </span><pre class="inline">sit amet'
-             '</pre>'),
+             'id., <span class="after_token">at</span> <span class="'
+             'after_token">123.</span> </span><pre class="inline">Lorem ipsum'
+             ' dolor sit amet</pre>'),
 
             # Id. citation across line break
             ('asdf." Id., at 315.\n       Lorem ipsum dolor sit amet',
-             '<pre class="inline">asdf." </pre><span class="citation '
-             'no-link">Id., at 315.\n       Lorem </span><pre class="inline">'
-             'ipsum dolor sit amet</pre>')
+             '<pre class="inline">asdf." </pre><span class="citation no-link">'
+             'Id., <span class="after_token">at</span> <span class="'
+             'after_token">315.</span>\n</span><pre class="inline">       '
+             'Lorem ipsum dolor sit amet</pre>'),
+
+            # Ibid. citation ("... Ibid.")
+            ('asdf, Ibid. Lorem ipsum dolor sit amet',
+             '<pre class="inline">asdf, </pre><span class="citation no-link">'
+             'Ibid. <span class="after_token">Lorem</span> <span class="'
+             'after_token">ipsum</span> <span class="after_token">dolor</span>'
```

Comment:
I'm surprised we touch anything after the Ibid, but perhaps this is your comment about needing to do this, which I didn't entirely understand, in which case, that's fine, if you say it's gotta be here.

Reply:
No, I was being obtuse again... Thanks for calling me out lol. I have (once again) removed these stupid "after token" spans, and replaced them with a single, simpler "id_token" span: c40b17f